mypy/types.py

```diff
@@ -2925,16 +2926,26 @@ def visit_type_var(self, typ: TypeVarType) -> Type:
         return typ
 
 
+class LocationSetter(TypeTraverserVisitor):
+    def __init__(self, line: int, column: int) -> None:
```

Comment:
Add a TODO comment along the lines of "Should we update locations of other Type subclasses?".

Reply:
added